pkg/types/ovirt/platform.go

```diff
@@ -33,4 +33,22 @@ type Platform struct {
 	// Default will set the image field to the latest RHCOS image.
 	// +optional
 	DefaultMachinePlatform *MachinePool `json:"defaultMachinePlatform,omitempty"`
+
+	// AffinityGroups contains the RHV affinity groups that the installer will create.
```

Comment:
Suggestion:

> AffinityGroups contains a list of affinity groups that will be created by the installer. If an affinity group is not listed here, but specified in AffinityGroupNames it is assumed that the affinity group already exists in oVirt.

Reply:
I don't want to add a reference to the machine field here, I think that saying it on the machine field is more than enough